public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -497,6 +497,13 @@ export class DashboardMigrator {
       });
     }
 
+    if (oldVersion < 23) {
+      if (getConfig().featureToggles.newVariables) {
+        this.dashboard.variables.list = cloneDeep(this.dashboard.templating.list);
+        this.dashboard.templating.list = [];
```

Comment:
Not sure about setting `this.dashboard.templating.list` to empty might be hard to debug errors. WDYT @mckn @torkelo.

Reply:
@hugohaggmark I think we should just delete it

Reply:
@dprokop reallly? Maybe we should do this at the same time we remove the featureToggle is my thinking?

Reply:
@hugohaggmark that's what I meant, sorry for not being clear